cli/replicaset/lua/cartridge/edit_topology_body.lua

```diff
@@ -0,0 +1,18 @@
+local cartridge = require('cartridge')
+
+local args, wait_rw, timeout = ...
+
+local res, err = cartridge.admin_edit_topology({
+    servers = args.servers,
+    replicasets = args.replicasets
+})
+
+if err ~= nil then
+    err = err.err
+end
+
+assert(err == nil, tostring(err))
+
+if wait_rw then
+    box.ctl.wait_rw(timeout)
```

Comment:
Do we need a little more clever wait here?
```suggestion
    box.ctl.wait_rw(timeout)
```

https://github.com/tarantool/cartridge-cli/blob/76044114f412b1fa15e353f84e7de1f0c3d98566/cli/replicasets/edit_topology.go#L103-L107

Reply:
I don't mind. How do you think, could we move this waiting to the server side, similarly to `box.ctl.wait_rw`?

Reply:
In general, I would answer yes. Only for `promote` the answer is not so obvious.

Reply:
Thank you! The logic is extended.